Template.php

```diff
@@ -1395,13 +1395,8 @@ protected function use_unnamed_params() {
           }
           if ($matched_parameter) {
             $dat = trim(str_replace($oMatch, "", $dat));
-            if ($i == 0) { // Use existing parameter slot in first instance
```

Comment:
The objective here is to minimise the change as far as possible, keeping the sequence position of the misspelt parameter, and the spacing as far as possible.